contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -124,13 +127,13 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils5, DaoOperator, ReentrancyGua
         updateBRRData(DEFAULT_REWARD_BPS, DEFAULT_REBATE_BPS, now, 0);
     }
 
-    modifier onlyDAO {
-        require(msg.sender == address(kyberDAO), "only DAO");
+    modifier onlyDao {
+        require(msg.sender == address(kyberDao), "only dao");
         _;
     }
 
     modifier onlyKyberNetwork {
-        require(msg.sender == address(kyberNetwork), "only Kyber");
+        require(msg.sender == address(kyberNetwork), "only network");
```

Comment:
should be `kyberNetwork` so it is same as comments and other revert messages?
Also should we use Dao or always kyberDao?

Reply:
yes, it should be the same. `only kyberNetwork`